src/overlays/actors/ovl_Dm_Bal/z_dm_bal.c

```diff
@@ -30,40 +30,41 @@ ActorInit Dm_Bal_InitVars = {
     (ActorFunc)DmBal_Draw,
 };
 
+// These animations are the same and in the same order as in the main NPC actor EnBal
 typedef enum {
-    /*  -1 */ DMBAL_ANIM_NONE = -1,
-    /* 0x0 */ DMBAL_ANIM_0,
-    /* 0x1 */ DMBAL_ANIM_1,
-    /* 0x2 */ DMBAL_ANIM_2,
-    /* 0x3 */ DMBAL_ANIM_3,
-    /* 0x4 */ DMBAL_ANIM_4,
-    /* 0x5 */ DMBAL_ANIM_5,
-    /* 0x6 */ DMBAL_ANIM_6,
-    /* 0x7 */ DMBAL_ANIM_7,
-    /* 0x8 */ DMBAL_ANIM_8,
-    /* 0x9 */ DMBAL_ANIM_9,
-    /* 0xA */ DMBAL_ANIM_10,
-    /* 0xB */ DMBAL_ANIM_11,
-    /* 0xC */ DMBAL_ANIM_12,
-    /* 0xD */ DMBAL_ANIM_13,
-    /* 0xE */ DMBAL_ANIM_MAX
-} DmBalAnimation;
-
-static AnimationInfo sAnimationInfo[DMBAL_ANIM_MAX] = {
-    { &object_bal_Anim_0005FC, 1.0f, 0.0f, 0.0f, ANIMMODE_LOOP, -8.0f },  // DMBAL_ANIM_0
-    { &object_bal_Anim_000840, 1.5f, 0.0f, 0.0f, ANIMMODE_LOOP, -8.0f },  // DMBAL_ANIM_1
-    { &object_bal_Anim_000840, 1.5f, 0.0f, 0.0f, ANIMMODE_ONCE, -4.0f },  // DMBAL_ANIM_2
-    { &object_bal_Anim_00A7DC, 1.0f, 0.0f, 0.0f, ANIMMODE_ONCE, -4.0f },  // DMBAL_ANIM_3
-    { &object_bal_Anim_00B1E8, 1.0f, 0.0f, 0.0f, ANIMMODE_LOOP, -8.0f },  // DMBAL_ANIM_4
-    { &object_bal_Anim_00B604, 1.0f, 0.0f, 0.0f, ANIMMODE_LOOP, -8.0f },  // DMBAL_ANIM_5
-    { &object_bal_Anim_00C498, 1.0f, 0.0f, 0.0f, ANIMMODE_ONCE, -8.0f },  // DMBAL_ANIM_6
-    { &object_bal_Anim_00C8D8, 1.0f, 0.0f, 0.0f, ANIMMODE_LOOP, -8.0f },  // DMBAL_ANIM_7
-    { &object_bal_Anim_00C8D8, 1.0f, 0.0f, 0.0f, ANIMMODE_ONCE, -8.0f },  // DMBAL_ANIM_8
-    { &object_bal_Anim_00C498, 1.0f, 23.0f, 0.0f, ANIMMODE_ONCE, -8.0f }, // DMBAL_ANIM_9
-    { &object_bal_Anim_00D530, 1.0f, 0.0f, 0.0f, ANIMMODE_LOOP, -8.0f },  // DMBAL_ANIM_10
-    { &object_bal_Anim_000C78, 1.0f, 0.0f, 0.0f, ANIMMODE_LOOP, -2.0f },  // DMBAL_ANIM_11
-    { &object_bal_Anim_00CB78, 1.0f, 0.0f, 0.0f, ANIMMODE_LOOP, -2.0f },  // DMBAL_ANIM_12
-    { &object_bal_Anim_001804, 1.0f, 0.0f, 0.0f, ANIMMODE_LOOP, -2.0f },  // DMBAL_ANIM_13
+    /*  -1 */ TINGLE_ANIM_NONE = -1,
+    /* 0x0 */ TINGLE_ANIM_FLOAT_IDLE,
```

Comment:
```suggestion
    /*  -1 */ TINGLE_CS_ANIM_NONE = -1,
    /* 0x0 */ TINGLE_CS_ANIM_FLOAT_IDLE,
```
It's not standardized yet but it's been something I've been thinking about, it might be good to add `CS` to this name to differentiate it from the other Tingle Anim enum.